src/core/localization/translations/en/common-messages.json

```diff
@@ -68,12 +68,12 @@
       "init": "Select Geometry",
       "loading": "Loading analytics...",
       "error": "Error",
-      "other": "Analytics",
+      "header_title": "Analytics",
```

Comment:
did you also regenerate Spanish?

Reply:
it's not in the main branch, need to add it.

Reply:
@IlyaIzr could you please approve it https://github.com/konturio/disaster-ninja-fe/pull/151 ? 
and then we can fix key in spanish too

Reply:
@IlyaIzr merged

Reply:
changed it in spanish file too, resolving this one